parm/atm/obs/config/lgetkf_amsua_n19.yaml

```diff
@@ -29,46 +29,46 @@ obs error:
 obs filters:
 - filter: Bounds Check
   filter variables:
-  - name: brightness_temperature
+  - name: brightnessTemperature
     channels: 4-6,9-14
   minvalue: 100.0
   maxvalue: 500.0
 - filter: Background Check
   filter variables:
-  - name: brightness_temperature
+  - name: brightnessTemperature
     channels: 4-6,9-14
   threshold: 3.0
 - filter: Domain Check
   filter variables:
-  - name: brightness_temperature
+  - name: brightnessTemperature
     channels: 4-6,9-14
   where:
   - variable:
-      name: scan_position@MetaData
+      name: sensorScanPosition@MetaData
```

Comment:
Should be MetaData/sensorScanPosition

Reply:
Thanks @ADCollard .   I'll make this change and rerun the ctests.